src/tests/CMakeLists.txt

```diff
@@ -185,3 +234,22 @@ if( MFEM_FOUND )
 
 endif()
 
+if( TRIBOL_USE_ENZYME )
+
+  set(enzyme_smoke_depends LLDEnzymeFlags gtest)
+  blt_add_executable(
+    NAME enzyme_smoke_test
+    SOURCES enzyme_smoke.cpp
+    OUTPUT_DIR ${TEST_OUTPUT_DIRECTORY}
+    DEPENDS_ON ${enzyme_smoke_depends} ${tribol_device_depends}
+    FOLDER tribol/tests )
+
+  blt_add_test( NAME enzyme_smoke_test
+                COMMAND enzyme_smoke_test )
+
+  if (ENABLE_CUDA)
```

Comment:
Can you clarify the need for the if (ENABLE_CUDA) and if (ENABLE_HIP) for enzyme tests? Enzyme is not ported to the gpu, correct? The enzyme tests won't run on device, but perhaps should work even for builds where CUDA or HIP are enabled?

Reply:
No, no GPU support for enzyme.  On CUDA and HIP builds, these lines are needed to link with the Tribol library.